lib/index.js

```diff
@@ -306,20 +315,26 @@ api.remove = async ({actor, account, type} = {}) => {
 /**
  * Verifies a token for an account.
  *
- * @param actor the actor or capabilities to perform the action.
- * @param [account] the ID of the account to verify a token for.
- * @param [email] the email of the account to verify a token for.
- * @param type the type of token to verify.
- * @param [hash] the bcrypt hash to verify for a password/nonce or other
- *   type of token that uses hashed token challenges.
- * @param [challenge] the token challenge value for token types that do not
- *   hash token challenges.
- * @param [authenticatedMethods=[]] a list of identifiers for other methods
- *   methods that have already been authenticated (useful for multifactor
- *   verification where one method depends on other methods to first be used).
+ * @param {object} options - The options to use.
+ * @param {object} options.actor - The actor or capabilities to perform the
+ *   action.
+ * @param {string} [options.account] - The ID of the account to verify a token
+ *   for.
+ * @param {string} [options.email] - The email of the account to verify a token
+ *   for.
+ * @param {string} options.type - The type of token to verify.
+ * @param {string} [options.hash] - The bcrypt hash to verify for a
+ *   password/nonce or other type of token that uses hashed token challenges.
+ * @param {string} [options.challenge] - The token challenge value for token
+ *   types that do not hash token challenges.
+ * @param {Array}[options.authenticatedMethods=[]] - A list of identifiers for
+ *   other methods methods that have already been authenticated (useful for
+ *   multifactor verification where one method depends on other methods to
+ *   first be used).
  *
- * @return a Promise that resolves to an object containing the account ID,
- *   the account `email` and token information if verified, `false` if not.
+ * @returns {Promise<object>} - A Promise that resolves to an object containing
+ *   the account ID,the account `email` and token information if verified,
```

Comment:
```suggestion
 *   the account ID, the account `email` and token information if verified,
```

Reply:
@mattcollier I will leave it to you if there should be a comma before the `and` in this sentence.